packages/genesys-spark-components/src/components/stable/gux-dropdown-multi/example.html

```diff
@@ -366,10 +366,23 @@ <h2>Unknown Value Set</h2>
   >
 
   <h2>Disabled</h2>
-  <gux-dropdown-multi value="a" disabled>
+  <gux-button
+    id="toggle-disabled-state"
+    onclick="(function() {
+      const dropdown = document.getElementById('toggle-disabled-state-dropdown');
+      const disabled = dropdown.getAttribute('disabled') === 'true';
+      dropdown.setAttribute('disabled', !disabled);
```

Comment:
This can be simplified to 
```
document.getElementById('toggle-disabled-state-dropdown').toggleAttribute('disabled');
```
https://developer.mozilla.org/en-US/docs/Web/API/Element/toggleAttribute

Reply:
Fixed